packages/orbit-components/src/primitives/IllustrationPrimitive/index.js

```diff
@@ -48,14 +48,22 @@ const IllustrationPrimitive = ({
   size = SIZE_OPTIONS.MEDIUM,
   dataTest,
   spaceAfter,
-}: Props) => (
-  <StyledImage
-    illustrationName={name}
-    alt={alt || name}
-    size={size}
-    data-test={dataTest}
-    spaceAfter={spaceAfter}
-  />
-);
+}: Props) => {
+  const getAltText = () => {
+    if (alt === "") return alt;
+    if (alt && alt.length > 0) return alt;
+    return name;
+  };
+
+  return (
+    <StyledImage
+      illustrationName={name}
+      alt={getAltText()}
```

Comment:
Looks like the logic could be simplified to this:

```jsx
alt={typeof alt === "string" ? alt : name}
```

It's absolutely unnecessary to hold this PR any further, I'm just saying this for educational purposes.